src/components/BRB/BRBMainComponent.tsx

```diff
@@ -665,6 +672,7 @@ const StyledHeader = styled.header`
 
 const NavText = styled.div`
   color: #fff;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family